CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Support for `AtomicCmpExchange128` intrinsic.
+- Support for `GetTypeKind` intrinsic.
+- Support for `OpenString` intrinsic type.
+
+### Changed
+
+- Improve `VarArg*` intrinsic signatures and make them stricter.
+
+### Fixed
+
+- The `IsManagedType` intrinsic would only accept type references - it now accepts values.
```

Comment:
This is phrased a little oddly - a user reading this probably wouldn't understand that this just means that `IsManagedType` invocations now correctly resolve when passed a value. Maybe

```suggestion
- Failure to resolve invocations of `System.IsManagedType` where a value is passed.
```